Gum/Commands/GuiCommands.cs

```diff
@@ -69,6 +69,12 @@ public void RefreshVariableValues()
             PropertyGridManager.Self.RefreshVariablesDataGridValues();
         }
 
+        public void ShiftAppFontSize(int delta)
+        {
+            MainWindow.MainMenuStrip.Font = new System.Drawing.Font(MainWindow.MainMenuStrip.Font.FontFamily, (float)mainPanelControl.FontSize - 2.75f + delta);
```

Comment:
I think I can answer that.  The font for the main app is 8.25 unedited (No idea why), so that 2.75 bumps it up to "11" which you set on line 44 of MainPanelConrol (But again, it's actually 8.25 default not 11).